demos/basic/main.cpp

```diff
@@ -33,12 +33,12 @@
 
 static std::string const testFile(TULPAR_DEMO_WORKING_DIR"data/ding_02.ogg");
 
-constexpr double pi()
+double pi()
```

Comment:
Would it be useful to have both versions available?

Reply:
Ditto